### DISCUSSION TITLE
为什么attu客户端中不能进行余弦相似度查询

### DISCUSSION THREAD

**Discussion by blessing59420** [milvus-io/milvus#27525] [Q&A and General discussion]
![image](https://github.com/milvus-io/milvus/assets/59166341/2f6d9bfa-011b-4dea-a279-c7a3a3d2e08d)
![image](https://github.com/milvus-io/milvus/assets/59166341/20d28147-86a2-444b-970a-dcc23e6722fb)
如图所示，选择不了cosine，只有L2和ip，我的索引是cosnie类型的

**Comment by yhmo** [comment]
The Attu v2.3.0 declares it supports cosine: https://github.com/zilliztech/attu/releases/tag/v2.3.0
If your attu version is lower than v2.3.0, upgrade to the latest version.
